src/client/src/apps/MainRoute/store/applicationServices.ts

```diff
@@ -13,12 +13,12 @@ import { User } from 'rt-types'
 import { ReplaySubject } from 'rxjs'
 import { retryWhen, multicast, refCount } from 'rxjs/operators'
 import { referenceDataService } from '../data/referenceData'
-import { LimitChecker, ExcelApp, PlatformAdapter } from 'rt-platforms'
```

Comment:
I removed this `PlatformAdapter` type in favor of simple `Platform` - I think users of this thing should not care if it's adapter or anything else